src/System.Globalization/tests/CultureInfo/CultureInfoClone.cs

```diff
@@ -25,5 +25,15 @@ public void Clone(CultureInfo culture)
             Assert.Equal(culture, clone);
             Assert.NotSame(clone, culture);
         }
+
+        [Fact]
+        public void TestCalendarAfterCloning()
+        {
+            CultureInfo ci = new CultureInfo("en-US");
+            Assert.True(ci.Calendar == ci.DateTimeFormat.Calendar);
```

Comment:
does `==` imply `object.ReferenceEqual` in this case?

Reply:
Yes == is reference equality for the calendar.